playback/src/player.rs

```diff
@@ -486,6 +486,7 @@ impl PlayerInternal {
         let track = Track::get(&self.session, track_id).wait().unwrap();
 
         info!("Loading track \"{}\"", track.name);
+        debug!("Spotify URI: spotify:track:{}", track_id.to_base62());
```

Comment:
Put these two on the same line

Reply:
You want me to combine them into an info or debug line?